src/Raven.Client/Documents/Operations/Revisions/RevertDocumentsToRevisionsOperation.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Collections.Generic;
+using Raven.Client.Documents.Commands;
+using Raven.Client.Documents.Conventions;
+using Raven.Client.Http;
+using Sparrow.Json;
+
+namespace Raven.Client.Documents.Operations.Revisions
+{
+    internal class RevertDocumentsToRevisionsOperation : IMaintenanceOperation
+    {
+        private readonly Dictionary<string, string> _idToChangeVector;
+
+        public RevertDocumentsToRevisionsOperation(Dictionary<string, string> idToChangeVector)
```

Comment:
also, we now need to add XML documentation for the exposed client API methods

Reply:
Done